examples/experimental-amp/pages/_app.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import App, { Container } from 'next/app'
+
+class MyApp extends App {
+  static async getInitialProps ({ Component, ctx }) {
+    let pageProps = {}
+
+    if (Component.getInitialProps) {
+      pageProps = await Component.getInitialProps(ctx)
+    }
+
+    return { pageProps }
+  }
+
+  render () {
+    const { Component, pageProps } = this.props
+
+    return (
+      <Container>
+        <Component {...pageProps} />
+        <style jsx global>{`
```

Comment:
Let's add the styles inside the page, possibly inside a `Layout` component.

Reply:
Sounds good, updated